lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -22,7 +27,7 @@ namespace Js
 
     RecyclableObject* JavascriptProxy::GetHandler()
     {
-        if (handler == nullptr)
+        if (IsRevoked(handler))
```

Comment:
Since `handler` and `target` will be `nullptr` for the revoked case, can this just be `IsRevoked()`?

Reply:
Yes, `IsRevoked()` will work as expect, I'll modify similar occurrences.

Reply:
Done.